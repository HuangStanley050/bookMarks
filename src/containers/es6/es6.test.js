import React from "react";
import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import ES6 from "./es6";
/*global expect*/

configure({ adapter: new Adapter() });

describe("ES6 Component testing", () => {
    let wrapper =

        beforeEach(() => {
            wrapper = shallow(<ES6/>);
        });
    it("should render one ES6 component", () => {

        expect(wrapper.find(ES6)).toHaveLength(1);
    });

});
