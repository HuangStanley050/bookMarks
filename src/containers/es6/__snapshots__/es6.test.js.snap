// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ES6 Component testing should render one ES6 component 1`] = `undefined`;
